### ISSUE TITLE
meeting place of barracks should be settable all over the map

### ISSUE THREAD

**Issue by codingberlin** [jsettlers/settlers-remake#298]
In distinction to e.g. wood cutters where it is good that the gathering point can't be set too far away it is not helpful that you can set the meeting point of the barrack only near by.

e.g. on maps like river or alien I would like to set the meeting point to my only entrance to my land in the middle game and later when I attack directly to my enemies land.

**Comment by DerKrawallkeks** [comment]
I thought the same thing. But here's the problem: do we really want to change all the little things that could be better than in the original? 
Another example: You could include a lot of statistics or messages that help you manage your economy a lot better.
But do we want that?

I don't even think that we shouldn't, but you should really think about it..

**Comment by michaelzangl** [comment]
This is one of the things we even intended to do: Make things better.

**Comment by krichter722** [comment]
> I thought the same thing. But here's the problem: do we really want to change all the little things that could be better than in the original?
Another example: You could include a lot of statistics or messages that help you manage your economy a lot better.
But do we want that?

In doubt, helpers can be implemented with an option which can be configured by the player or the developers when there's need for fine-tuning/balancing. Currently (before the first release) I wouldn't bother implementing as many great tools as come to mind.

**Comment by andreas-eberle** [comment]
@DerKrawallkeks: Yes, we really want to improve the game, so this issue is definitely in. Stuff like extended statistics could be added as well, but like @krichter722 said, stuff like that is far down the priority list.

**Comment by Ne-X-uS** [comment]
Is it a possibility to enhance the settings menu to disable or enable certain features?
Perhaps then both people are happy.

**Comment by PaweX** [comment]
But then on the other hand, In the original S3 you had to observe your colony for potential issues with your economy like: I see that my vineyard doesn't produce much wine, therefore I think I put it in wrong place. But when we are being informed about every little thing then:
1. Too many messages (to me it was already annoying in the original S3)
2. We become lazy about taking care of our economy because the game will tell us everything so we don't need to check anything.

**Comment by krichter722** [comment]
@PaweX A compromise between "nothing at all" and "too many" of anything is making things configurable - not always a good option for a game, but in this case an option with 1 to 3 degrees between any imaginable message and none is a good solution.

**Comment by PaweX** [comment]
Yes. Personally I would use the solution from Settlers IV where we can switch off specific messages. Also it would be nice if we can set in options to choose between classic (from original S3) way of messages appear and the new one.
